pyvista/plotting/volume.py

```diff
@@ -0,0 +1,65 @@
+"""PyVista volume module."""
+import pyvista as pv
+
+from ._property import Property
+from .mapper import _BaseMapper
+from .prop3D import Prop3D
+
+
+class Volume(Prop3D, pv._vtk.vtkVolume):
```

Comment:
Similar `_vtk` namespace concern.

We have to be carefule with dual inheritance with these VTK C++ classes. I think we need to make a base `Prop3D` class that defines all of the functionality that this then inherits.

Reply:
I don't see a proper solution without the use multiple inheritance to this. The problem is that `vtkProp3D` is an abstract class, and its abstract methods are implemented in `vtkVolume` in this case. 

A possible solution that avoids multiple inheritance is to wrap those methods in `pyvista.Volume()` but I think it would defeat the original purpose of the `pyvista.Prop3D` class, which is to evade duplicated code in `Volume()` and `Actor()` and simplify the code.

We can also just remove the `pyvista.Prop3D()` class and have its methods duplicated in both `Actor()` and `Volume()`, but I wouldn't prefer that solution, only if the dual inheritance is problematic enough to consider it.

In this case, I would argue that this dual inheritance shouldn't be problematic. We are having an inheritance from a abstract class and it's implementation, so all accessible methods should refer to the same functionality in the end.

If you think removing the dual inheritance is a must, let me know and we can figure out some solution.

Reply:
> I think we need to make a base Prop3D class that defines all of the functionality that this then inherits.

I agree with @AlejandroFernandezLuces. Code works as-is. I don't think we need a base `Prop3D` class and can proceed as-is.